cime_config/tests.py

```diff
@@ -427,38 +427,45 @@
     },
 
     "e3sm_scream" : {
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v0"),
+    ),
+
+    "e3sm_scream_v0" : {
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v0_lowres"),
+    ),
+
+    "e3sm_scream_v0_lowres" : {
         "time"  : "03:00:00",
         "tests" : (
             "SMS_D.ne4pg2_ne4pg2.F2010-SCREAM-HR",
             "SMS_D.ne4pg2_ne4pg2.F2010-SCREAM-LR",
-            "ERS.ne4pg2_ne4pg2.F2010-SCREAM-HR",
-            "ERS.ne4pg2_ne4pg2.F2010-SCREAM-LR",
             "ERP.ne4pg2_ne4pg2.F2010-SCREAM-HR.eam-double_memleak_tol",
             "ERP.ne4pg2_ne4pg2.F2010-SCREAM-LR.eam-double_memleak_tol",
-            "REP.ne4pg2_ne4pg2.F2010-SCREAM-HR",
-            "REP.ne4pg2_ne4pg2.F2010-SCREAM-LR",
-            "PEM.ne4pg2_ne4pg2.F2010-SCREAM-HR",
-            "PEM.ne4pg2_ne4pg2.F2010-SCREAM-LR",
-            "SMS_R_Ln10.ne4_ne4.FDPSCREAMA97",
-            "ERS_R_Ln10.ne4_ne4.FDPSCREAMA97",
             "ERP_R_Ln10.ne4_ne4.FDPSCREAMA97",
             )
     },
 
     "e3sm_scream_v1" : {
-        "time"  : "03:00:00",
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v1_lowres", "e3sm_scream_v1_medres"),
```

Comment:
Does this mean that both lowres and medres get run on all machines now? We were previously only running medres on some machs?

Reply:
I am planning to use this testsuite in the nightlies, but run only lowres in the AT run. But I forgot to add that switch in our jenkins script...